From 5eac0ee74944fc599e10fd25cec0f52cc655efe7 Mon Sep 17 00:00:00 2001
From: xWASABI <wasabi.dev@gmail.com>
Date: Tue, 12 May 2015 03:35:19 -0500
Subject: [PATCH 09/10] KIO - Exposed Settings Background ** Minor fix to allow
 better compatibility with other themes not utilizing the settings exposed
 background .xmls in their themes. ** Changed @color/ reference in
 drawable/dashboard_background_exposed.xml and
 drawable/settings_main_background_exposed.xml ** Please add this commit as
 well, cc45d78, for full exposing of settings backgrounds.

---
 res/drawable/dashboard_background_exposed.xml     | 2 +-
 res/drawable/settings_main_background_exposed.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/drawable/dashboard_background_exposed.xml b/res/drawable/dashboard_background_exposed.xml
index c346af8..5f8c186 100644
--- a/res/drawable/dashboard_background_exposed.xml
+++ b/res/drawable/dashboard_background_exposed.xml
@@ -4,7 +4,7 @@
     <item>
         <shape  android:shape="rectangle">
 
-            <solid android:color="@color/dashboard_background_color" />
+            <solid android:color="@color/exposed_bg_dark" />
 
         </shape>
     </item>
diff --git a/res/drawable/settings_main_background_exposed.xml b/res/drawable/settings_main_background_exposed.xml
index ee6b75a..23c0dfa 100644
--- a/res/drawable/settings_main_background_exposed.xml
+++ b/res/drawable/settings_main_background_exposed.xml
@@ -2,7 +2,7 @@
     <item>
         <shape  android:shape="rectangle">
 
-            <solid android:color="@color/exposed_bg_dark" />
+            <solid android:color="@color/dashboard_background_color" />
 
         </shape>
     </item>
-- 
1.9.1

